packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -135,6 +136,15 @@ export interface IContainerRuntimeBase extends
      */
     getQuorum(): IQuorum;
 
+    /**
+     * @deprecated This method is provided as a migration tool for customers currently reading the code details
+     * from within the Container by directly accessing the Quorum proposals.  The code details should not be accessed
+     * from within the Container as this requires coupling between the container contents and the code loader.
+     * Direct access to Quorum proposals will be removed in an upcoming release, and in a further future release this
+     * migration tool will be removed.
+     */
+    getSpecifiedCodeDetails?(): IFluidCodeDetails | undefined;
```

Comment:
i'm looking here

Reply:
so this will be removed?

Reply:
For context, we were discussing that the runtime should really never have access to the specified code details, which I agree with.  I think we probably can sufficiently cover this scenario without adding this API here by ensuring the config is "stamped" on the container in some way (e.g. create the correct type of component in current flow, store as data in the generic root component in future flow).  I'm investigating this further to validate.

Reply:
Removed in latest.